src/components/BoardUsers/BoardUser.tsx

```diff
@@ -0,0 +1,23 @@
+import './BoardUser.scss';
+import {getInitials} from 'constants/Name'
+export interface BoardUserProps {
+  id: string;
+  name: string;
+  avatar?: string;
+}
+
+const BoardUser = ({ name, avatar }: BoardUserProps) => {
```

Comment:
I would rename the component to `UserAvatar`, as we probably will have more components related to a user (e.g. `UserProfile` or something like that if we want to display more information)

Reply:
done